ckanext/canada/view.py

```diff
@@ -948,8 +949,15 @@ def _log_api_access(request_data, pkg_dict):
 
 
 def notice_no_access():
-    '''flash_notice if logged-in user can't actually do anything yet'''
-    if h.check_access('package_create'):
+    '''flash_notice if logged-in user can't actually do anything yet.'''
+    has_org_membership = model.Session.query(model.Member.id).filter(
+                            and_(model.Member.state == 'active',
+                                 model.Member.table_name == 'user',
+                                 model.Member.table_id == g.userobj.id,
+                                 or_(model.Member.capacity == 'admin',
+                                     model.Member.capacity == 'editor',
+                                     model.Member.capacity == 'member'))).first()
```

Comment:
I don't think there *are* any other capacities so this part doesn't do anything.